pkg/kubelet/kubelet.go

```diff
@@ -2437,6 +2406,8 @@ func (kl *Kubelet) syncLoopIteration(configCh <-chan kubetypes.PodUpdate, handle
 		}
 		kl.addSource(u.Source)
 
+		kl.handleMirrorPods(&u)
```

Comment:
Make it clear that the update list may be modified with a comment, e.g.: "Handle mirror pods separately and remove them from the list"